cumulusci/salesforce_api/filterable_objects.py

```diff
@@ -0,0 +1,261 @@
+NOT_COUNTABLE = (
+    "ContentDocumentLink",  # ContentDocumentLink requires a filter by a single Id on ContentDocumentId or LinkedEntityId
+    "ContentFolderItem",  # Implementation restriction: ContentFolderItem requires a filter by Id or ParentContentFolderId
+    "ContentFolder",  # Similar to above
+    "ContentFolderLink",  # Similar to above
+    "ContentFolderMember",  # Similar to above
+    "IdeaComment",  # you must filter using the following syntax: CommunityId = [single ID],
+    "Vote",  # you must filter using the following syntax: ParentId = [single ID],
+    "RecordActionHistory",  # Gack: 1133111327-118855 (1126216936)
+)
+
+
+NOT_EXTRACTABLE = NOT_COUNTABLE + (
+    "%__Share",
```

Comment:
Assuming these regexes are applied using a whole-string match, I think we can just use

```suggestion
    "%Share",
```

and remove the specific entries below. Share objects always end in `Share` for standard objects, and `__Share` for custom objects.